lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py

```diff
@@ -3,61 +3,149 @@
 
 Sample usage::
 
-    python lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py --app-csv "edx-platform Apps Ownership.csv"
+    python lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py --repo-csv "Individual Repo Ownership.csv" --app-csv "edx-platform Apps Ownership.csv" --dep-csv "edx-platform 3rd-party Ownership.csv"
 
-Sample CSV input::
+Or for more details::
 
-    Path,owner.squad
-    ./common/djangoapps/xblock_django,team-red
-    ./openedx/core/djangoapps/xblock,team-red
-    ./lms/djangoapps/badges,team-blue
+    python lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py --help
 
-Sample output::
-
-    # Copy results into appropriate config yml file.
-    CODE_OWNER_MAPPINGS:
-      team-blue:
-      - badges
-      team-red:
-      - xblock_django
-      - openedx.core.djangoapps.xblock
 
 """
-import os
-import re
 import csv
 import click
+import os
+import re
+
+# Maps edx Django apps to their containing repo
+EDX_REPO_APPS = {
+    'bulk_grades': 'edx-bulk-grades',
+    'coaching': 'platform-plugin-coaching',
+    'completion': 'completion',
+    'config_models': 'django-config-models',
+    'consent': 'edx-enterprise',
+    'csrf': 'edx-drf-extensions',
+    'edx_proctoring': 'edx-proctoring',
+    'edxval': 'edx-val',
+    'enterprise': 'edx-enterprise',
+    'enterprise_learner_portal': 'edx-enterprise',
+    'help_tokens': 'help-tokens',
+    'integrated_channels': 'edx-enterprise',
+    'organizations': 'edx-organizations',
+    'search': 'edx-search',
+    'wiki': 'django-wiki',
+}
+
+# Maps third-party Django apps to their containing repo
```

Comment:
This says "their containing repo" but I'm a little confused about the context. Are these repos in various Github orgs? Other places?

Reply:
Let me know if it is more clear now.